ports/stm32/boards/stm32f401xe.ld

```diff
@@ -9,7 +9,8 @@ MEMORY
     FLASH_ISR (rx)  : ORIGIN = 0x08000000, LENGTH = 16K /* sector 0 */
     FLASH_FS (rx)   : ORIGIN = 0x08004000, LENGTH = 112K /* sectors 1,2,3 are 16K, 4 is 64K */
     FLASH_TEXT (rx) : ORIGIN = 0x08020000, LENGTH = 384K /* sectors 5,6,7 are 128K */
-    RAM (xrw)       : ORIGIN = 0x20000000, LENGTH = 96K
+    FS_CACHE (xrw)  : ORIGIN = 0x20000000, LENGTH = 16K
+    RAM (xrw)       : ORIGIN = 0x20004000, LENGTH = 80K
```

Comment:
I somehow feel it would be slightly better to put FS_CACHE after RAM... at least so when you inspect RAM via a debugger things start at 0x20000000.

Reply:
Okay, updated.